packages/tocco-ui/src/EditableValue/typeEditors/DateAbstract.js

```diff
@@ -131,25 +131,37 @@ class DateAbstract extends React.Component {
 
   render() {
     return (
-      <StyledDateAbstractWrapper
-        data-wrap
-        onBlur={this.handleOnBlur}
-        immutable={this.props.immutable}
-        ref={this.wrapper}
+      <div
         id={this.props.id}
+        tabIndex="-1"
+        onFocus={() => {
+          const inputElement = this.wrapper.current.getElementsByClassName('flatpickr-input')[1]
```

Comment:
this selection seems quite unstable and might break with a future flatpickr update